### PR TITLE
PMM-2433, PMM-2434: Use new options, SlowLogRotation and RetainSlowLogs

### PR DESCRIPTION
**Pull request by arvenil**
Use new options: SlowLogRotation and RetainSlowLogs.
Those are sent from pmm-client to qan-agent through qan-api: https://github.com/percona/pmm-client/pull/129

### CODE REVIEW

File: qan/analyzer/mysql/config/config.go [hunk 1]
```diff
@@ -27,11 +27,12 @@ import (
 )
 
 var (
-	DEFAULT_INTERVAL              uint  = 60         // 1 minute
-	DEFAULT_MAX_SLOW_LOG_SIZE     int64 = 1073741824 // 1G
-	DEFAULT_REMOVE_OLD_SLOW_LOGS        = true       // whether to remove old slow logs after rotation
-	DEFAULT_OLD_SLOW_LOGS_TO_KEEP       = 1          // how many slow logs to keep on filesystem
-	DEFAULT_EXAMPLE_QUERIES             = true
+	DEFAULT_INTERVAL             uint  = 60         // 1 minute
```
Comment (golangci) [inline qan/analyzer/mysql/config/config.go]: don't use ALL_CAPS in Go names; use CamelCase

File: qan/analyzer/mysql/config/config.go [hunk 2]
```diff
@@ -27,11 +27,12 @@ import (
...
+	DEFAULT_MAX_SLOW_LOG_SIZE    int64 = 1073741824 // 1G
```
Comment (golangci) [inline qan/analyzer/mysql/config/config.go]: don't use ALL_CAPS in Go names; use CamelCase

File: qan/analyzer/mysql/config/config.go [hunk 3]
```diff
@@ -27,11 +27,12 @@ import (
...
+	DEFAULT_SLOW_LOGS_ROTATION         = true       // whether to rotate slow logs
```
Comment (golangci) [inline qan/analyzer/mysql/config/config.go]: don't use ALL_CAPS in Go names; use CamelCase

File: qan/analyzer/mysql/config/config.go [hunk 4]
```diff
@@ -27,11 +27,12 @@ import (
...
+	DEFAULT_REMOVE_OLD_SLOW_LOGS       = true       // whether to remove old slow logs after rotation
```
Comment (golangci) [inline qan/analyzer/mysql/config/config.go]: don't use ALL_CAPS in Go names; use CamelCase

File: qan/analyzer/mysql/config/config.go [hunk 5]
```diff
@@ -27,11 +27,12 @@ import (
...
+	DEFAULT_SLOW_LOGS_TO_KEEP          = 1          // how many slow logs to keep on filesystem
```
Comment (golangci) [inline qan/analyzer/mysql/config/config.go]: don't use ALL_CAPS in Go names; use CamelCase

File: qan/analyzer/mysql/config/config.go [hunk 6]
```diff
@@ -27,11 +27,12 @@ import (
...
+	DEFAULT_EXAMPLE_QUERIES            = true
```
Comment (golangci) [inline qan/analyzer/mysql/config/config.go]: don't use ALL_CAPS in Go names; use CamelCase

**Review by askomorokhov — APPROVED** [review]
Looks good to me. 👍

**Review by percona-csalguero — APPROVED** [review]

**Pull request merged**